### ISSUE TITLE
NTR: Food at birth

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB-ontology/VEuPathDB-ontology#510]
For new term requests, please provide the following information:
**Term Label** [required]
Food at birth

**Synonyms** [optional]

**Definition** [required except for organism name]
The definition should be understandable even for non-specialists. Include a PubMed ID or DOI to refer to any relevant article that provides additional information about the suggested term.

What type of food the participant consumed at birth (breast milk, top milk, sugar water, etc.)

**Suggested Parent Term** [required]
Please look in the ontology hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/eupath) or choose one from following list: 
- [ ] Material (independent of other entities and can be maintained through time, such as person, cell, sample)
- [ ] Process (has temporal proper parts and for some time depends on some material entity, such as life cycle, cell division, heart beating, sleeping)
- [ ] Information (generically dependent on some artifact and stands in relation of aboutness to some entity, such as data, identifier, document)
- [ ] Characteristic/Quality (inheres in or is borne by other entities, such as height, color, disease)

Information

**Examples** [optional]

**Contributor** [optional]
If you would like to be a contributor of the term or anyone you think that contributes to the term, such as label, definition, please provide the name and ORCID ID.

**Requester** [optional]
Please indicate which resource/database/project/ request the term.
ClinEpiDB

**Comment by johnwjudkins** [comment]
@SheenaTomko 'Food at birth' EUPATH:0060139 and 'Food since birth' EUPATH:0036412 are used in the same study (Crypto). I'm guessing "food at birth" means the same day, probably within hours. Is that correct?

**Comment by SheenaTomko** [comment]
@johnwjudkins yes. Food at birth indicates what they were given within an hour or so of birth. Food since birth was collected repeatedly over a period of months when they are likely to still be breast feeding but may be consuming formula instead

**Comment by johnwjudkins** [comment]
@SheenaTomko 
EUPATH:0060139
Label: information on food fed to neonate at birth
Definition: An information content entity that specifies food products, including water, fed to a neonate within approximately an hour after its birth.
Parent: information content entity
Subclass axiom: mentions some 'food product'

**Comment by SheenaTomko** [comment]
Looks good, thanks @johnwjudkins

**Comment by zhengj2007** [comment]
Looks good to me.
